Khan/perseus
# [UnlimitedPolygon] Fix open polygon scoring issues.

## Summary:
This PR fixes and adds 3 things to the Unlimited Polygon Experience:

1. Only closed unlimited polygons are scorable.
2. Correct Unlimited Polygons in our editor are closed when rendered (i.e. unlimited polygons that are rendered with exisitng coordinates of 3 or more are closed at start).
3. Added save warnings to editor to notify content creators to close their unlimited polygons before saving.

Issue: LEMS-2682

## Test plan:

1. Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--interactive-graph-unlimited-polygon.
2. Notice that the unlimited polygon correct answer is automatically closed on render.
3. Go to http://localhost:6006/?path=/story/perseus-widgets-interactive-graph--unlimited-polygon-with-mafs
4. Draw a polygon and check it when it's open, you should see the score is type `invalid`
5. Close the polygon and git it again, it should be a `valid` score.
